### PR TITLE
Add days flag to to stats cli

### PR DESCRIPTION
**Pull request by mannytoledo**
This addresses issue #2

### CODE REVIEW

File: bin/stats [hunk 1]
```diff
@@ -9,8 +9,8 @@ import datetime
 from clint.textui import puts, colored
 from clint.textui import columns
 
-sys.path.append("../Trellodog")
-
+dir = os.path.dirname(__file__)
+sys.path.append(os.path.join(dir,"../Trellodog"))
 
 VERSION = "0.1"
```
Comment (chrishenry) [inline bin/stats]: Maybe a version bump?

File: bin/stats [hunk 2]
```diff
@@ -161,6 +161,7 @@ if __name__ == "__main__":
     from trellodog import Trellodog
     from optparse import OptionParser
     parser = OptionParser(version=VERSION, usage=usage)
+    parser.add_option("-d", "--days", action="store", type="int", dest="days", help="number of days of activity to return")
```
Comment (chrishenry) [inline bin/stats]: Let's set a default here. Maybe 3, or 7?

Comment (mannytoledo) [inline bin/stats]: Default is still set on the activity method in the `trellodog.py` module. [This](https://github.com/mannytoledo/trellodog/blob/master/Trellodog/trellodog.py#L59) should default it to 3 days.

Comment (chrishenry) [inline bin/stats]: Right, but there's no way to expose that via CLI. Setting a default here, even if it's the same, means we can communicate it in help messages, etc.

For DRYness sake, you can make the default a class variable of the Trellodog class, and reference it both here, and in the method signature.

**Comment by chrishenry** [comment]
:+1:

**Pull request merged**